assets/src/blocks/Spreadsheet/SpreadsheetFrontend.js

```diff
@@ -61,62 +61,61 @@ export class SpreadsheetFrontend extends Component {
       const baseUrl = document.body.dataset.nro;
 
       const spreadSheetData = baseUrl
-        ? await fetchJson(`${ baseUrl }/wp-json/${ addQueryArgs('planet4/v1/get-spreadsheet-data', args) }`)
-        : await apiFetch({ path: addQueryArgs('planet4/v1/get-spreadsheet-data', args) });
+        ? await fetchJson( `${ baseUrl }/wp-json/${ addQueryArgs( 'planet4/v1/get-spreadsheet-data', args ) }` )
+        : await apiFetch( { path: addQueryArgs( 'planet4/v1/get-spreadsheet-data', args ) } );
 
-      this.setState({loading:false, spreadSheetData})
+      this.setState( { loading: false, spreadSheetData } );
     } else {
       if ( this.props.handleErrors ) {
         this.props.handleErrors( { invalidSheetId: true } );
       }
-      this.setState({ loading: false, spreadSheetData: null });
+      this.setState( { loading: false, spreadSheetData: null } );
     }
   }
 
   async componentDidMount() {
-    if (this.props.url !== '') {
-      await this.fetchSheetData(this.props.url);
+    if ( this.props.url !== '' ) {
+      await this.fetchSheetData( this.props.url );
     }
   }
 
-  async componentWillReceiveProps(nextProps) {
-    if (nextProps.url !== this.props.url) {
-      await this.fetchSheetData(nextProps.url);
+  async UNSAFE_componentWillReceiveProps( nextProps ) {
```

Comment:
This also causes a console warning, but I guess it would need a full refactor to fix so probably ok for now 🤷‍♀️